packages/components/src/components/Callout/Callout.tsx

```diff
@@ -0,0 +1,155 @@
+import { SystemProp, Theme } from "@xstyled/styled-components";
+import React from "react";
+import { Box } from "../../primitives/Box";
+import { Icon, IconProps } from "../Icon";
+
+type CalloutVariantOptions = "error" | "information" | "success" | "warning";
+
+type CalloutProps = {
+  /** The content of the callout. It can be text or valid text related HTML, i.e. anchor and strong elements. */
+  children: React.ReactNode;
+
+  /** The type of callout. */
+  variant?: CalloutVariantOptions;
+
+  /** Show or hide the callout background */
+  withBackground?: boolean;
+};
+
+const getVariantProps = (
+  variant: CalloutVariantOptions,
+  withBackground: boolean
+): {
+  backgroundColor?: SystemProp<keyof Theme["colors"], Theme>;
+} => {
+  if (!withBackground) {
+    return {};
+  }
+
+  switch (variant) {
+    case "error": {
+      return {
+        backgroundColor: "colorBackgroundError",
+      };
+    }
+    case "warning": {
+      return {
+        backgroundColor: "colorBackgroundWarning",
+      };
+    }
+    case "success": {
+      return {
+        backgroundColor: "colorBackgroundSuccess",
+      };
+    }
+    default: {
+      return {
+        backgroundColor: "colorBackgroundInfo",
+      };
+    }
+  }
+};
+
+const getIconVariantProps = (
+  variant: CalloutVariantOptions
+): {
+  color: SystemProp<keyof Theme["colors"], Theme>;
+  icon: IconProps["icon"];
+} => {
+  switch (variant) {
+    case "error": {
+      return {
+        color: "colorIconError",
+        icon: "XCircleIcon",
+      };
+    }
+    case "warning": {
+      return {
+        color: "colorIconWarning",
+        icon: "ExclamationTriangleIcon",
```

Comment:
Can you also add the title of the Icon, please?

Reply:
The `decorative` on the icon adds `aria-label=true`, so we don't need an `aria-label` in this case.